src/number/n_GF.jl

```diff
@@ -93,19 +93,34 @@ function show(io::IO, c::N_GField)
    print(io, "Finite Field of Characteristic ", characteristic(c), " and degree ", degree(c))
 end
 
+function Base.show(io::IO, ::MIME"text/plain", a::n_GF)
+  print(io, AbstractAlgebra.obj_to_string(a, context = io))
+end
+
+function AbstractAlgebra.expressify(a::n_GF; context = nothing)::Any
+  F = parent(a)
+  i = reinterpret(Int, a.ptr.cpp_object)
+  if 1 < i < characteristic(F)^degree(F)
```

Comment:
Can this overflow?

Reply:
no

Reply:
see https://github.com/oscar-system/Singular.jl/blob/master/src/number/n_GF.jl#L390